src/components/game-board/board.tsx

```diff
@@ -16,6 +16,11 @@ export const MyContext = createContext("my-context");
 export const GameBoard = component$(() => {
   useStylesScoped$(styles);
 
+  const answer = Array.from({ length: 4 }, () => {
+    const randomColor = Math.floor(Math.random() * 6);
+    return ["Q", "W", "E", "R", "T", "Y"][randomColor];
```

Comment:
abstract this "possible colors" array out of this function. prob should live in the index.tsx of the board route